tools/klee/main.cpp

```diff
@@ -1118,13 +1117,13 @@ linkWithUclibc(StringRef libDir, std::string opt_suffix,
 }
 
 int main(int argc, char **argv, char **envp) {
-  atexit(llvm_shutdown);  // Call llvm_shutdown() on exit.
+  atexit(llvm_shutdown); // Call llvm_shutdown() on exit
 
-#if LLVM_VERSION_CODE >= LLVM_VERSION(13, 0)
-  KCommandLine::HideOptions(llvm::cl::getGeneralCategory());
-#else
-  KCommandLine::HideOptions(llvm::cl::GeneralCategory);
-#endif
+  KCommandLine::KeepOnlyCategories(
+     { &ChecksCat,      &DebugCat,      &ExtCallsCat, &ExprCat,   &LinkCat,
+      &MemoryCat,      &MergeCat,    &MiscCat,     &ModuleCat, &ReplayCat,
+      &SearchCat,      &SeedingCat,  &SolvingCat,  &StartCat,  &StatsCat,
+      &TerminationCat, &TestCaseCat, &TestGenCat});
```

Comment:
It looks like there is an attempt at fancy formatting, but it is not anymore? (Maybe the opening curly was originally on the previous line, and then moved?)

Reply:
Good catch.  For some reason, clang-format does align the elements here, although it does if you declare a variable and initialise it with that set...  I fixed it now.